main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
```

Comment:
Could start loop at 0 to avoid having to add one more iteration in next line to get a total of 10.